<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMain</class>
 <widget class="QMainWindow" name="QMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Labyrinth2015</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Menu">
    <property name="title">
     <string>&amp;Menu</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionInvite_players"/>
    <addaction name="actionNew_game"/>
    <addaction name="actionLoad_game"/>
    <addaction name="actionSave_game"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="separator"/>
    <addaction name="actionDebug"/>
   </widget>
   <addaction name="menu_Menu"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="separator"/>
   <addaction name="actionInvite_players"/>
   <addaction name="separator"/>
   <addaction name="actionNew_game"/>
   <addaction name="actionLoad_game"/>
   <addaction name="actionSave_game"/>
   <addaction name="separator"/>
   <addaction name="action_About"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="action_About">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/gfx/about.png</normaloff>:/gfx/about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionInvite_players">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/gfx/invite.png</normaloff>:/gfx/invite.png</iconset>
   </property>
   <property name="text">
    <string>Invite players</string>
   </property>
  </action>
  <action name="actionNew_game">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/gfx/new.png</normaloff>:/gfx/new.png</iconset>
   </property>
   <property name="text">
    <string>New game</string>
   </property>
  </action>
  <action name="actionLoad_game">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/gfx/load.png</normaloff>:/gfx/load.png</iconset>
   </property>
   <property name="text">
    <string>Load game</string>
   </property>
  </action>
  <action name="actionSave_game">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/gfx/save.png</normaloff>:/gfx/save.png</iconset>
   </property>
   <property name="text">
    <string>Save game</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/gfx/connect.png</normaloff>:/gfx/connect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/gfx/disconnect.png</normaloff>:/gfx/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>QMain</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_About</sender>
   <signal>triggered()</signal>
   <receiver>QMain</receiver>
   <slot>handleShowHelp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionInvite_players</sender>
   <signal>triggered()</signal>
   <receiver>QMain</receiver>
   <slot>handleShowInvite()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoad_game</sender>
   <signal>triggered()</signal>
   <receiver>QMain</receiver>
   <slot>handleShowLoadGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_game</sender>
   <signal>triggered()</signal>
   <receiver>QMain</receiver>
   <slot>handleShowSaveGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew_game</sender>
   <signal>triggered()</signal>
   <receiver>QMain</receiver>
   <slot>handleShowNewGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionConnect</sender>
   <signal>triggered()</signal>
   <receiver>QMain</receiver>
   <slot>handleShowConnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDisconnect</sender>
   <signal>triggered()</signal>
   <receiver>QMain</receiver>
   <slot>handleDisconnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDebug</sender>
   <signal>triggered()</signal>
   <receiver>QMain</receiver>
   <slot>debugToggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>handleShowHelp()</slot>
  <slot>handleShowInvite()</slot>
  <slot>handleShowNewGame()</slot>
  <slot>handleShowLoadGame()</slot>
  <slot>handleShowSaveGame()</slot>
  <slot>handleShowConnect()</slot>
  <slot>handleDisconnect()</slot>
  <slot>debugToggle()</slot>
 </slots>
</ui>
